File: Results/DER/Translation/translation_output/misleading_test/codellama/codenet/java/python-sanitized/p01388.py
import math

def run():
    s = input().toCharArray()
    c = [0, 0, 0, 0]
    for i in range(len(s)):
        if s[i] == 'K':
            c[0] += 1
        elif s[i] == 'U':
            c[1] += 1
        elif s[i] == 'P':
            c[2] += 1
        elif s[i] == 'C':
            c[3] += 1
    print(math.min(c[0], math.min(c[1], math.min(c[2], c[3]))))

if __name__ == '__main__':
    run()

File: Results/DER/Translation/translation_output/no_test/wizardcoder/codenet/python/java/p01388.java
Below is an instruction that describes a task. Write a response that appropriately completes the request.

### Instruction:
Translate the following python code to java and enclose your solution inside ```java```:
```
a=input()
K=a.count('K')
U=a.count('U')
P=a.count('P')
C=a.count('C')
print(min(K,U,P,C))

```

### Response:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a = sc.nextLine();
        int K = 0, U = 0, P = 0, C = 0;
        for (int i = 0; i < a.length(); i++) {
            char ch = a.charAt(i);
            if (ch == 'K') {
                K++;
            } else if (ch == 'U') {
                U++;
            } else if (ch == 'P') {
                P++;
            } else if (ch == 'C') {
                C++;
            }
        }
        System.out.println(Math.min(K, Math.min(U, Math.min(P, C))));
    }
}
```<|endoftext|>
